PendingReleaseNotes

```diff
@@ -173,10 +173,11 @@
   users to view the availability score for each pool in a cluster. A pool is considered 
   unavailable if any PG in the pool is not in active state or if there are unfound 
   objects. Otherwise the pool is considered available. The score is updated every 
-  5 seconds. The feature is on by default. A new config option ``enable_availability_tracking``
-  can be used to turn off the feature if required. Another command is added to clear the 
-  availability status for a specific pool, ``ceph osd pool clear-availability-status <pool-name>``. 
-  This feature is in tech preview. 
+  1 second by default. This interval can bee changed using the new config option
+  ``pool_availability_update_intervl.``. The feature is on by default. A new config option 
+  ``enable_availability_tracking`` can be used to turn off the feature if required. 
+  Another command is added to clear the availability status for a specific pool, 
+  ``ceph osd pool clear-availability-status <pool-name>``. This feature is in tech preview. 
```

Comment:
Suggest the last sentence in a new paragraph so it doesn't seem like it only applies to "clear-availability-status"